ui/desktop/app/services/store.js

```diff
@@ -3,7 +3,11 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import Store, { CacheHandler } from '@ember-data/store';
+// Importing 'Store' from 'ember-data/store' to get default configuration automatically.
+// TODO: Remove eslint-disable after upgrading to eslint-plugin-ember >= v12.2.1
```

Comment:
none blocking note: when we perform the ember upgrade within desktop, we will update `eslint-plugin-ember` to latest and came back here for the fix 😉

Reply:
awesome! that works out then!